doc/schemas/pm-context-schema.json

```diff
@@ -21,13 +22,16 @@
         "contextGroupName": {
           "type": "string"
         },
+        "comment": {
+          "type": "string"
+        },
```

Comment:
not sure about that field. I added it because there are two comments in the context files and validation fails if the definition here doesn't exist